.github/workflows/commentOnNativeFileChanged.yml

```diff
@@ -0,0 +1,27 @@
+name: Comment on native files changed
+
+on:
+  pull_request:
+    branches: 
+      - 'main'
+    paths:
+      - '**.kt'
+      - '**.java'
+      - '**.swift'
+      - '**.mm'
+      - '**.h'
+      - '**.cpp'
+      - 'AndroidManifest.xml'
+      - 'project.pbxproj'
+
+jobs:
+  commentOnNativeFilesChanged:
+    name: Create comment
+    runs-on: ubuntu-latest
+    steps: 
+      - name: Comment
+        run: |
+          gh pr comment ${{ github.event.number }} --body \
+          ":warning: This PR may change native code. It may cause problems in Hybrid App. Please ask Hybrid App team to review those changes. :warning:"
```

Comment:
```suggestion
          ":warning: This PR is possibly changing native code. It may cause problems in Hybrid App. Please ask Hybrid App team to review those changes in the Slack open-source channel. The C+ can help you with that. :warning:"
```